homeassistant/components/unifi/unifi_entity_base.py

```diff
@@ -102,35 +101,10 @@ async def remove_item(self, keys: set) -> None:
             entity_registry.async_remove(self.entity_id)
             return
 
-        if (
```

Comment:
@Kane610 I've removed the logic for maybe deleting devices. The device registry will automatically delete devices if no config entries are linked so this logic shouldn't be needed.

Reply:
@balloob I think the same apply not to Fritz: https://github.com/home-assistant/core/blob/dev/homeassistant/components/fritz/common.py#L459-L477